lib/receptacle/test_support.rb

```diff
@@ -47,21 +47,5 @@ def with_strategy(strategy, repo = described_class)
     ensure
       repo.strategy original_strategy
     end
-
-    # ensure_method_delegators
-    #
-    # When using something like `allow(SomeRepo).to receive(:find)` (where find
-    # is a mediated method) before the method was called the first time Rspec
-    # would stub the method with the wrong arity of 0 when also using jruby.
-    # This seems to be caused by the lazily defined methods. Simply calling the
-    # following method in a before hook when method stubbing is need solves this
-    # issue. As it's a problem which only affects mocking libraries it's
-    # currently not planned to change this behavior.
-    #
-    def ensure_method_delegators(repo)
-      Receptacle::Registration.repositories[repo].methods.each do |method_name|
-        repo.__build_method(method_name)
-      end
-    end
```

Comment:
Is this method removed without any substitution?
If so please add a note to the upgrade notes.

Reply:
yes its not needed anymore. will add to the upgrade notes :+1:

Reply:
hmm ok but i think this will not help as this will only pass when that bug is released, which will be 9.4.1.0